core/dbt/compilation.py

```diff
@@ -246,7 +246,7 @@ def _inject_ctes_into_sql(self, sql: str, ctes: List[InjectedCTE]) -> str:
         return str(parsed)
 
     def _get_dbt_test_name(self) -> str:
-        return 'dbt__CTE__INTERNAL_test'
+        return 'dbt__cte__internal_test'
```

Comment:
This should probably not be hard-coded within the compiler, and should instead do something like `adapter.add_ephemeral_prefix('internal_test')`, but that's a problem for another day. We're planning to clean up some inconsistencies around data tests for the v0.20.0 release, anyway.

Reply:
That would be nice to see 🙂